### ISSUE TITLE
gem metadat not shown on rubydoc.info

### ISSUE THREAD

**Issue by hramrach** [docmeta/rubydoc.info#189]
Looking at https://www.rubydoc.info/gems/lmsensors

Where is it from?

Does it have a web page?

Who wrote it?

When?

There is nothing.

All of that is in the gem metadata.

**Comment by lsegal** [comment]
All gems hosted by rubydoc are available as rubygems from rubygems.org/gems/GEMNAME, in this case: https://rubygems.org/gems/lmsensors

I think it could make sense to link to rubygems in the breadcrumb.

**Comment by hramrach** [comment]
Yes, reusing the rubygems website that already has the information makes sense.

**Comment by lsegal** [comment]
Closing out some old issues that have likely been resolved since the original report. If this is still a problem please reopen, thanks!

(Note: this is a bulk message, apologies in advance if this feels impersonal)
